src/resources/authoring/gv2.json

```diff
@@ -283,6 +283,7 @@
         "alleles": "w-W, m-m, T-T, h-h, A1-A1, C-C, B-B, D-D, rh-rh, Bog-Bog",
         "sex": 0
       },
+      "gametes": [[{}, {}, {}, {}, {}, {}, {}, {}], [{}, {}, {}, {}, {}, {}, {}, {}]],
```

Comment:
I did give it some thought and probably would have changed it if not for the feeling that Trudi's going to want to have more control over the set of gametes generated, and so we're likely to end up with a similar-looking array of more fully-specified gametes whose order gets randomized at run-time. But there's no guarantee of that, and I agree that long strings of `{}`s in a row should not be the API we end up with.